pages/en/lb3/Controlling-data-access.md

```diff
@@ -355,3 +355,165 @@ To upgrade an application from an earlier version of LoopBack 3.x, follow these
 Specify a `DEBUG` environment variable with value `loopback:security:*` for the console to log the lookups and checks the server
 makes as requests come in, useful to understand things from its perspective.
 Do this in your test environment as there may be quite a lot of output.
+
+## Comprehensive AccessType, Property and End-point
+<table>
+  <tbody>
+    <tr>
+      <th>Access Type</th>
+      <th>Property</th>
+      <th>Verb</th>
+      <th>End point</th>
+    </tr>
+    <tr>
+      <td>READ</td>
+      <td>find</td>
+      <td>GET</td>
+      <td>/model-name-plural</td>
+    </tr>
+    <tr>
+      <td>&nbsp;</td>
+      <td>findById</td>
+      <td>GET</td>
+      <td>/model-name-plural/{id}</td>
+    </tr>
+    <tr>
+      <td>&nbsp;</td>
+      <td>exists</td>
```

Comment:
hi @frankorama , there are two `exists`

Reply:
I just checked in the loopback explorer and there are in fact 2 end-points that map to the `exists` property as described in my documentation